atomic_reactor/schemas/config.json

```diff
@@ -734,6 +734,18 @@
         "additionalProperties": false,
         "required": ["name", "value"]
       }
+    },
+    "image_size_limitation": {
+      "description": "Define maximum image size for regular image checked before push",
```

Comment:
It would be nice to document the unit of size (is it bytes?)

Reply:
And mention it's uncompressed image size.

Reply:
Done